include/ur_modern_driver/ros/lowbandwidth_trajectory_follower.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <inttypes.h>
+#include <array>
+#include <atomic>
+#include <cstddef>
+#include <cstring>
+#include <string>
+#include <thread>
+#include <vector>
+#include "ur_modern_driver/log.h"
+#include "ur_modern_driver/ur/commander.h"
+#include "ur_modern_driver/ur/server.h"
+#include "ur_modern_driver/ros/trajectory_follower_interface.h"
+
+class LowBandwidthTrajectoryFollower:  public TrajectoryFollowerInterface
+{
+private:
+  std::atomic<bool> running_;
+  std::array<double, 6> last_positions_;
+  URCommander &commander_;
+  URServer server_;
+
+  double time_interval_, servoj_time_, servoj_time_waiting_, max_waiting_time_, \
+         servoj_gain_, servoj_lookahead_time_, max_joint_difference_;
+  bool debug_, more_debug_;
+
+  std::string program_;
+
+  template <typename T>
+  size_t append(uint8_t *buffer, T &val)
+  {
+    size_t s = sizeof(T);
+    std::memcpy(buffer, &val, s);
+    return s;
+  }
```

Comment:
Well. When I started LBTF I wanted to gain minmal impact so I tried to avoid commonalising code or creating some common util file (and I am also a proponent of making "util" kind of classes/methods when they are needed for the 3rd time - not the second), but in this case it's even easier, because in fact LBTF does not use append method any more :). So I simply removed it in https://github.com/Zagitta/ur_modern_driver/pull/9/commits/992c661e9e65435bcede95f5521259f58c614ef1